ndsl/grid/eta.py

```diff
@@ -30,10 +30,8 @@ class HybridPressureCoefficients:
 
 
 def _load_ak_bk_from_file(eta_file: str) -> Tuple[np.ndarray, np.ndarray]:
-    if eta_file == "None":
-        raise ValueError("eta file not specified")
     if not os.path.isfile(eta_file):
-        raise ValueError("file " + eta_file + " does not exist")
+        raise ValueError("eta file does not exist")
```

Comment:
Keep `eta_file` in there, probably make a f-string to be more inline with the rest of the code